agent/unit-test/softwarecontaineragent_unittest.cpp

```diff
@@ -24,28 +24,78 @@
 #include <gtest/gtest.h>
 #include <unistd.h>
 
-class CreateContainerTest: public ::testing::Test
+class SoftwareContainerAgentTest: public ::testing::Test
 {
 public:
-    CreateContainerTest() { }
+    SoftwareContainerAgentTest() { }
     SoftwareContainerAgent *sca;
     Glib::RefPtr<Glib::MainContext> m_context = Glib::MainContext::get_default();
     int m_preloadCount = 1;
     bool m_shutdownContainers = true;
     int m_shutdownTimeout = 2;
+    SoftwareContainerWorkspace workspace;
 
     void SetUp() override
     {
         sca = new SoftwareContainerAgent(m_context
                                          , m_preloadCount
                                          , m_shutdownContainers
                                          , m_shutdownTimeout);
+
+        workspace = sca->getSoftwareContainerWorkspace();
     }
 
 };
 
-TEST_F(CreateContainerTest, TestActivateWithNoConf) {
-    //ASSERT_FALSE(gw->activate());
-    ASSERT_EQ(true,true);
+TEST_F(SoftwareContainerAgentTest, CreateContainerWithNoConf) {
+    ContainerID id = sca->createContainer("iejr-", "");
+    // This is actually only true if no other containers have been created
+    // before this one. Might need to be fixed somehow.
+    ASSERT_TRUE(id == 0);
+}
+
+TEST_F(SoftwareContainerAgentTest, CreateContainerWithConf) {
+    ContainerID id = sca->createContainer("iejr-", "[{\"writeOften\": \"1\"}]");
```

Comment:
Should it really be "writeOften": "1" instead of "writeOften": true? JSON supports true/false, so we should use them imho.